src/app/components/SearchFilter/SearchFilter.tsx

```diff
@@ -12,6 +11,8 @@ import {
 } from '@patternfly/react-core';
 import SearchIcon from '@patternfly/react-icons/dist/js/icons/search-icon';
 
+import { useTranslation } from '@rhoas/app-services-ui-components';
```

Comment:
Fixing the hook import will solve the issue on the missing translation.

To check this is working you have to run `cos-ui` in federated mode and open it inside `app-services-ui`. 
The issue is not there when running `cos-ui` as standalone.